assets/js/src/newsletters/send/congratulate/congratulate.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MailPoet from 'mailpoet';
+import moment from 'moment';
+
+import Success from './success.jsx';
+import Fail from './fail.jsx';
+import Loading from './loading.jsx';
+
+const SECONDS_WAITING_FOR_SUCCESS = 20;
+
+function close() {
+  return MailPoet.Ajax.post({
+    api_version: window.mailpoet_api_version,
+    endpoint: 'settings',
+    action: 'set',
+    data: { show_congratulate_after_first_newsletter: false },
+  }).always(() => {
+    window.location = window.mailpoet_main_page;
+  });
+}
+
+function renderSuccess(newsletter, testingPassed) {
+  if (testingPassed) {
+    MailPoet.trackEvent('Cron testing done', {
+      'Cron is working': 'true',
+    });
+  }
+  return (<Success
+    illustrationImageUrl={window.mailpoet_congratulations_success_image}
+    successClicked={close}
+    newsletter={newsletter}
+  />);
+}
+
+function renderFail() {
+  MailPoet.trackEvent('Cron testing done', {
+    'Cron is working': 'false',
+  });
+  return (<Fail
+    failClicked={close}
```

Comment:
Do we want to mark congratulation screen as viewed for failed use-case? I would suggest just to redirect to main page.

I think it is mentioned in spec:
`Important: If the cron previously did not work, ensure that the user goes through this verification process again.`